exampleAppKotlin/build.gradle

```diff
@@ -33,19 +33,20 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:multidex:1.0.3'
-    compile 'com.android.support:customtabs:27.1.1'
-    testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:27.1.1'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:multidex:1.0.3'
+    implementation 'com.android.support:customtabs:27.1.1'
+    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.anko:anko:$anko_version"
```

Comment:
What is this library for?

Reply:
https://github.com/Kotlin/anko, I'm planning the use this library when further improvement to the app is done. it can be safely removed though if not needed.

Reply:
Ok, if currently there are nothing using it, can you remove it?